src/main/java/Ui.java

```diff
@@ -0,0 +1,45 @@
+import task.Task;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+
+    private Scanner sc;
+    private static final String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
```

Comment:
## Note that:
- static variables should be of `UPPER_CASE_SNAKE_CASE`
- static class variables should be declared before non-static variables
```suggestion
    private static final String LOGO = " ____        _        \n"
            + "|  _ \\ _   _| | _____ \n"
            + "| | | | | | | |/ / _ \\\n"
            + "| |_| | |_| |   <  __/\n"
            + "|____/ \\__,_|_|\\_\\___|\n";
     private Scanner sc;
```